Resources/Prototypes/_Goobstation/Entities/Markers/Spawners/Random/landmines.yml

```
@@ -0,0 +1,18 @@
+    - type: Sprite
+      layers:
+        - state: red
+        - sprite: Objects/Misc/landmine.rsi
+          state: landmine
+    - type: RandomSpawner
+      chance: 0.90
+      prototypes:
+        - LandMineModular
+      rareChance: 0.10
+      rarePrototypes:
+      - LandMineExplosive
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)